interpreter/cling/lib/Interpreter/DeclUnloader.cpp

```diff
@@ -603,6 +603,14 @@ namespace cling {
     // * variables and parameters with dependent context;
     // * mangled names for parameters;
     if (!isa<ParmVarDecl>(VD) && !VD->getDeclContext()->isDependentContext()) {
+      // Exception variables without identifiers are not added to scope and will
+      // fail when tried to unload.
+      if (!VD->getIdentifier() && VD->isExceptionVariable()) {
```

Comment:
Isn’t that rule true for non-exception variables, too?

Reply:
I can't find or imagine a case where non-exception `VarDecl`s are unnamed. So, I am not sure.

Reply:
Perhaps we can drop the second check or make it an assert?